<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XImageEditClass</class>
 <widget class="QWidget" name="XImageEditClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XImageEdit</string>
  </property>
  <widget class="QPushButton" name="openB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>761</width>
     <height>781</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="XImage" name="image">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>759</width>
      <height>779</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="penButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>画笔</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="eraserButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>橡皮擦</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="rectButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>矩形</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="undoButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="redoButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>480</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="penSizeSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XImage</class>
   <extends>QWidget</extends>
   <header location="global">XImage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="XImageEdit.qrc"/>
 </resources>
 <connections/>
</ui>
